doc/snippets/Microsoft.Data.SqlClient/SqlException.xml

```diff
@@ -153,6 +153,10 @@ catch (Exception ex) {
             </remarks>
             <exception cref="T:System.ArgumentNullException">The <paramref name="si" /> parameter is a null reference (<see langword="Nothing" /> in Visual Basic).</exception>
         </GetObjectData>
+        <IsTransient>
+            <summary>Indicates whether the error represented by this <see cref="T:Microsoft.Data.SqlClient.SqlException" /> could be a transient error, i.e. if retrying the triggering operation may succeed without any other change. Examples of transient errors include failure to acquire a database lock, networking issues. This allows automatic retry execution strategies to be developed without knowledge of specific database error codes.</summary>
+            <returns><see langword="true"/> if the error represented could be a transient error; <see langword="true"/> otherwise.</returns>
```

Comment:
```suggestion
            <returns><see langword="true"/> if the error represented could be a transient error; <see langword="false"/> otherwise.</returns>
```